store.rb

```diff
@@ -0,0 +1,46 @@
+require 'json'
+require './filechecker'
+
+# Stores data in json files
+class Storing
+  def stores_data(arr, path)
+    data = JSON.parse(FileChecker.read_json_file(path))
```

Comment:
- [ ] When trying  to add a person in case the persons.json file doesn't exsist the user receives a type error as you can see below, this is the same for the books and the rentals.

![image](https://github.com/Yonashenok/school_library/assets/39240820/2639ede9-4a0c-4756-87f4-638040d93206)